beta/iam/guides/how-to-messaging-delegate.md

```diff
@@ -326,7 +326,30 @@ func shouldShowMessage(message: Showable) -> Bool {
 
 {% tab title="Android" %}
 
-Feature not yet available.
+If the developer would like to manually integrate the `View` that contains the UI for an in-app message, they can do so by accessing the `WebView` directly in a `MessagingDelegate` method.  
+
+In the example below, the developer decides whether or not the in-app message should be directly integrated into their existing UI.  If so, they capture a reference to the message's `WebView` and return `false` to prevent the message from being shown by the SDK:
```

Comment:
"In the example below, you can decide whether or not the in-app message should be directly integrated into your existing UI. If so, you capture a reference..."